operators/advanced_network/advanced_network/managers/gpunetio/CMakeLists.txt

```diff
@@ -16,31 +16,56 @@ cmake_minimum_required(VERSION 3.20)
 
 message(STATUS "PROJECT_NAME: ${PROJECT_NAME}")
 
-target_include_directories(${PROJECT_NAME} PRIVATE ${CMAKE_CURRENT_SOURCE_DIR})
-target_sources(${PROJECT_NAME} PRIVATE
-  adv_network_doca_mgr.cpp
-  adv_network_doca_mgr_obj.cpp
-  adv_network_doca_kernels.cu
+# --- Dependencies ---
+
+pkg_check_modules(DOCA REQUIRED doca-gpunetio doca-eth doca-flow)
+pkg_check_modules(DPDK REQUIRED libdpdk)
+
+# --- Sources ---
+
+target_sources(${PROJECT_NAME}
+  PRIVATE
+    adv_network_doca_mgr.cpp
+    adv_network_doca_mgr_obj.cpp
+    adv_network_doca_kernels.cu
 )
 
-pkg_check_modules(DOCA REQUIRED doca-gpunetio)
-pkg_check_modules(DOCA REQUIRED doca-flow)
-pkg_check_modules(DOCA REQUIRED doca-common)
-pkg_check_modules(DOCA REQUIRED doca-argp)
+# --- Compile settings ---
 
-pkg_check_modules(DPDK REQUIRED libdpdk)
+target_include_directories(${PROJECT_NAME}
+  PUBLIC ${DPDK_INCLUDE_DIRS}
+  PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}
+)
 
-target_include_directories(${PROJECT_NAME} PUBLIC ${DPDK_INCLUDE_DIRS})
-target_compile_options(${PROJECT_NAME}  PUBLIC ${DPDK_CFLAGS})
+target_compile_options(${PROJECT_NAME}
+  PUBLIC
+    ${DPDK_CFLAGS}
+    ${DOCA_CFLAGS}
+)
 
-target_compile_definitions(${PROJECT_NAME} PUBLIC ALLOW_EXPERIMENTAL_API)
-target_compile_definitions(${PROJECT_NAME} PUBLIC DOCA_ALLOW_EXPERIMENTAL_API)
+target_compile_definitions(${PROJECT_NAME}
+  PUBLIC
+    ALLOW_EXPERIMENTAL_API
+    DOCA_ALLOW_EXPERIMENTAL_API
+)
 
-set_target_properties(${PROJECT_NAME} PROPERTIES CUDA_SEPARABLE_COMPILATION ON)
-set_target_properties(${PROJECT_NAME} PROPERTIES CUDA_RESOLVE_DEVICE_SYMBOLS ON)
-set_target_properties(${PROJECT_NAME} PROPERTIES CUDA_ARCHITECTURES "80;90")
+set_target_properties(${PROJECT_NAME}
+  PROPERTIES
+    CUDA_SEPARABLE_COMPILATION ON
```

Comment:
this is what turns rdc on. I missed this before but I'd rather not have this enabled.

Reply:
that's just for gpunetio though, not propagated otherwise. also, this commit doens't add it, it just makes the cmakelists more readable and removes the flags not needed.

Agreed we can discuss rdc, but that's not in this MR's scope.